carsFilter/src/main/java/tkhal.filter.controller/CarsFilter.java

```diff
@@ -4,17 +4,25 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @author Taras Khalak
+ */
 public class CarsFilter {
-
+    /**
+     * This is the class for filtering cars
+     */
     private final static Logger LOGGER = LoggerFactory.getLogger(MessageFilterController.class);
 
-    public Car filter(Car tempCar) {
-        // filtering car
+    public Boolean filter(Car tempCar) {
```

Comment:
why not to use primitive?
Keep in mind: prefer using primitives when it possible

Reply:
done